command_interpreter:
  ros__parameters:
    base_url: "http://localhost:11434/v1"
    model: "llama3.2"
    EXTRACT_DATA_SERVICE_NAME: "/extract_data"
